Mastodon.xcworkspace/xcshareddata/IDETemplateMacros.plist

```diff
@@ -0,0 +1,3 @@
+{
+	FILEHEADER = " Copyright © ___YEAR___ Mastodon gGmbH. All rights reserved.";
```

Comment:
❤️  so much 👍

Reply:
@kimar We could cherrypick it to develop? So others (and you) can profit from it, too?